File: sdk/java/src/main/java/com/lightswitch/domain/Property.java
package com.lightswitch.domain;

public class Property {

	private String property;
	private String data;

	public Property(String property, String data) {
		this.property = property;
		this.data = data;
	}

	public String getProperty() {
		return property;
	}

	public String getData() {
		return data;
	}
}

File: backend/core-service/src/main/kotlin/com/lightswitch/core/domain/flag/repository/entity/Property.kt
package com.lightswitch.core.domain.flag.repository.entity

import com.lightswitch.core.common.entity.BaseEntity
import com.lightswitch.core.domain.history.dto.PreProperty
import jakarta.persistence.*

@Entity(name = "property")
class Property(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val propertyId: Long? = null,

    var property: String,
    var data: String,

    @JoinColumn(name = "keyword_id")
    @ManyToOne(fetch = FetchType.LAZY)
    val keyword: Keyword,
) : BaseEntity(){

    fun toPrevious(): PreProperty {
        return PreProperty(this)
    }
}